bitcoinplusplus/layer-2/clara-shikhelman:-unjamming-lightning.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Clara Shikhelman: Unjamming Lightning"
```

Comment:
There is no need to add the name of the speaker in the title, we already have it as metadata